internal/providers/terraform/azure/notification_hubs.go

```diff
@@ -0,0 +1,111 @@
+package azure
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/infracost/infracost/internal/usage"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetAzureRMNotificationHubsRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_notification_hub_namespace",
+		RFunc: NewAzureRMNotificationHubs,
+	}
+}
+
+func NewAzureRMNotificationHubs(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var monthlyAdditionalPushes *decimal.Decimal
+	sku := "Basic"
+	location := d.Get("location").String()
+
+	if d.Get("sku_name").Type != gjson.Null {
+		sku = d.Get("sku_name").String()
+	}
+	costComponents := make([]*schema.CostComponent, 0)
+	costComponents = append(costComponents, notificationHubsCostComponent("Namespace usage", location, sku))
+	if u != nil && u.Get("monthly_additional_pushes").Type != gjson.Null {
+		monthlyAdditionalPushes = decimalPtr(decimal.NewFromInt(u.Get("monthly_additional_pushes").Int()))
+		if monthlyAdditionalPushes.GreaterThanOrEqual(decimal.NewFromInt(10000000)) {
+			if sku == "Basic" {
+				costComponents = append(costComponents, notificationHubsPushesCostComponent("Additional pushes (Over 10M)", location, sku, "10", monthlyAdditionalPushes, 10000000))
+			}
+			if sku == "Standard" && monthlyAdditionalPushes.GreaterThan(decimal.NewFromInt(10000000)) {
+				pushLimits := []int{10000000, 100000000}
+				pushQuantities := usage.CalculateTierBuckets(*monthlyAdditionalPushes, pushLimits)
+				if pushQuantities[1].GreaterThanOrEqual(decimal.NewFromInt(10000000)) {
+					newPushes := &pushQuantities[1]
+					if pushQuantities[1].GreaterThanOrEqual(decimal.NewFromInt(100000000)) {
+						remainingPushes := pushQuantities[1].Sub(pushQuantities[0])
+						newPushes = &remainingPushes
+					}
+					costComponents = append(costComponents, notificationHubsPushesCostComponent("Additional pushes (10-100M)", location, sku, "10", newPushes, 1000000))
+				}
+				if pushQuantities[2].GreaterThanOrEqual(decimal.NewFromInt(10000000)) {
```

Comment:
This should be 
```if pushQuantities[2].GreaterThan(decimal.Zero) {```
and the quantity pass to `notificationHubsPushesCostComponent` should be `pushQuantities[2]`